README.md

```diff
@@ -1,3 +1,108 @@
-# Eclipse SmartHome Packaging Sample
+# Eclipse Smarthome Packaging Sample
+<img src="https://www.eclipse.org/concierge/images/logo.png" alt="concierge osgi" width="200"/>
 
-This repo contains a sample of how to create a small working runtime package that uses the Eclipse SmartHome framework.
+This repository contains a sample project which wraps eclipse smarthome within a concierge OSGi container, a lightweight OSGi R5 implementation. You can use this example to build an own minimal distribution with a very optimized memory footprint.
+
+More information about concierge: https://www.eclipse.org/concierge/index.php
+
+1. Prerequisites - Install Maven
```

Comment:
I had exactly this issue as JavaSE 7 was my default JVM.
@kaikreuzer should this maybe even be checked in start script? (start.sh)